workshops/pixel_art_pad/README.md

```diff
@@ -0,0 +1,456 @@
+---
+name: 'Pixel Art Pad'
+description: 'Build a pixel art pad using HTML and Java'
+author: '@hackyguru'
+img: 'https://cloud-2qbyb4hwx.vercel.app/0pixelpad.png'
+---
+
+# Pixel Art Pad
+
+Most of us like pixel art. In this workshop, I'm going to show you how to make your own pixel art pad with which you can create some amazing pixel art works.
+
+The pixel art pad will be looking like this (You can consider modifying the look if you wish) :
+
+[![homepage](https://cloud-2qbyb4hwx.vercel.app/0pixelpad.png)](https://pixel-creator-2.19eucs071kumara.repl.co/)
+
+You can also take a look at the [live demo][final_live_demo] and [final code][final_code].
+
+[final_live_demo]: https://pixel-creator-2.19eucs071kumara.repl.co/
+[final_code]: https://repl.it/@19EUCS071KUMARA/Pixel-Art-Pad
+[repl_it]: https://repl.it
+
+## Part 1: Prerequisites
+
+It is recommended to have a basic understanding of:
+
+- HTML
+- Java
+
+We will be using a library called processing.js which would help us in processing the graphics in a web canvas. All the functionalities of the pixel art pad will be written on java.
+
+However, this workshop is beginner friendly and you can refer the [final code][final_code] to understand better.
+
+
+## Part 2: Getting started
+
+### Setting up Repl.it
+
+[Repl.it](https://repl.it) is an amazing online code editor where we will writing our code. Though repl.it is not mandatory, I strongly recommend you to use it in order to avoid installations and downloads.
```

Comment:
```suggestion
[Repl.it](https://repl.it) is an amazing online code editor where we will be writing our code. Though repl.it is not mandatory, I strongly recommend you to use it in order to avoid installations and downloads.
```